pkgs/applications/video/avidemux/default.nix

```diff
@@ -57,7 +57,12 @@ stdenv.mkDerivation rec {
 
   buildCommand = let
     wrapWith = makeWrapper: filename:
-      "${makeWrapper} ${filename} --set ADM_ROOT_DIR $out --prefix LD_LIBRARY_PATH : ${libXext}/lib";
+      ''
+        ${makeWrapper} ${filename} \
+          --set ADM_ROOT_DIR $out \
+          --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libXext (placeholder "out") ]} \
+          --prefix LD_LIBRARY_PATH : $out/lib
```

Comment:
```suggestion
          --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ libXext (placeholder "out") "${placeholder "out"}/lib"]}
```

Reply:
> `makeLibraryPath [ /* ... */ "${placeholder "out"}/lib" ]

That doesn't seem right.

Reply:
I understand why my first attempt at applying @Ma27's suggestion is wrong now: having `placeholder "out"` as in the list passed to `makeLibraryPath` already does what the second `--prefix` is trying to do.

But I agree, [`makeLibraryPath`'s documentation](https://github.com/NixOS/nixpkgs/blob/master/lib/strings.nix#L198) says it appends the `/lib` already, so _also_ having `${placeholder "out"}/lib` in the list seems like it'll generate a `/lib/lib` path, which doesn't seem right.

Anyway, third attempt just pushed.  I confirmed it builds and launches cleanly.